src/domain/RootViewModel.js

```diff
@@ -77,7 +81,20 @@ export class RootViewModel extends ViewModel {
         } else if (loginToken) {
             this.urlCreator.normalizeUrl();
             if (this.activeSection !== "login") {
-                this._showLogin(loginToken);
+                this._showLogin({ loginToken });
+            }
+        } else if (oidcError) {
+            this._setSection(() => this._error = new Error(`OIDC error: ${oidcError[1]}`));
```

Comment:
Ideally, we'd show this error in a dedicated screen, like for SSO, see CompleteSSOLoginViewModel